utils.py

```diff
@@ -0,0 +1,10 @@
+import logging
+
+
+def get_logger_by_name(logger_name: str):
```

Comment:
You can provide the default value as a root, `logger_name: str = "root"` but I think it's better not to use default logger everywhere, just use `__name__` for the logger name when other modules call this function.

```python
from utils import get_logger_by_name

logger = get_logger_by_name(logger_name=__name__)

logger.debug(msg="TEST")
```